custom_components/myskoda/coordinator.py

```diff
@@ -96,6 +96,7 @@ def __init__(
         self.update_air_conditioning = self._debounce(self._update_air_conditioning)
         self.update_vehicle = self._debounce(self._update_vehicle)
         self.update_positions = self._debounce(self._update_positions)
+        self.duration: float | None = None
```

Comment:
Please store this in an appropriate section of `self.data` and update classes accordingly

Reply:
created additional config class in self.data so we can store also other stuff if needed in future